### ISSUE TITLE
p-toggleSwitch Accessibility Issues

### ISSUE THREAD

**Issue by geromegrignon** [openng-org/optimus-ui#351]
### Describe the bug

The p-toggleSwitch component with checkboxes has accessibility violations when running the IBM Equal Accessibility Checker. The two issues are: Each form control must have an associated label and UI component must have aria attributes.

### Reason for not contributing a PR

- [x] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/stackblitz-starters-dltq5smk

### Environment

stackblitz

### Angular version

20.0.4

### PrimeNG version

v20

### Steps to reproduce the behavior

1. Install the [IBM Equal Accessibility Checker](https://chromewebstore.google.com/detail/ibm-equal-access-accessib/lkcagbfjnkomcinoddgooolagloogehp)
2. Open the stackblitz
3. Run the accessibility checker
4. Filter on violations only and note the errors: 'Form control with "switch" role has no associated label' and 'The ARIA attributes "aria-checked" are not valid for the element <input> with ARIA role "switch"'

### Expected behavior

There are no accessibility violations for the p-toggleSwitch component when toggleSwitch is  checked/used

> Migrated from `primefaces/primeng#18766` — originally by `@srujana10499` on 2025-08-15

**Comment by geromegrignon** [comment]
*[Originally posted by `@github-actions[bot]` on 2026-02-12T00:27:16.000Z]*

This issue has been automatically marked as stale due to a lack of activity. **If this issue is still relevant to you, please leave a comment** so we can keep it open. We apologize for not being able to prioritize it sooner. If you have any new information or questions, please share them in your comment!
